bin/release

```diff
@@ -116,9 +121,11 @@ async function makeRelease() {
 		cp( 'dist/assets' ),
 		cp( 'includes' ),
 		cp( 'third-party' ),
-	] );
+	].concat(
+		'production' !== flagMode ? cp( 'dist/config.json' ) : []
+	) );
```

Comment:
Is the empty array here correct? Maybe this should rather be `undefined`? If the `flagMode` isn't "production" then we don't need to concatenate anything.

Alternatively we could make this a bit more intuitive to read and put the array of promises into a variable, and then add the additional entry only if needed, rather than always concatenating (potentially with nothing).

Reply:
Yes, if you concatenate anything other than an array, it appends that to the array, otherwise it joins the two arrays into a new one. We did the same thing before with some of the `Select` options I recall, but no problem to change. I'll update 👍 

![image](https://user-images.githubusercontent.com/1621608/101409917-3f90ff00-38e7-11eb-92c0-b295a27a6843.png)